cmd/amass/db.go

```diff
@@ -0,0 +1,162 @@
+// Copyright 2017 Jeff Foley. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bytes"
+	"errors"
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/OWASP/Amass/amass/core"
+	"github.com/OWASP/Amass/amass/handlers"
+	"github.com/OWASP/Amass/amass/utils"
+	"github.com/fatih/color"
+)
+
+const (
+	dbUsageMsg = "db [-d domain] [-dir PATH] [-config PATH] [-list] [-i optsFile]"
+)
+
+type dbArgs struct {
+	Domains utils.ParseStrings
+	Options struct {
+		ListEnumerations bool
+	}
+	Filepaths struct {
+		ConfigFile string
+		Directory  string
+		Domains    string
+		Input      string
+	}
+}
+
+func runDBCommand(clArgs []string) {
+	var args dbArgs
+	var help1, help2 bool
+	dbCommand := flag.NewFlagSet("db", flag.ExitOnError)
+
+	dbBuf := new(bytes.Buffer)
+	dbCommand.SetOutput(dbBuf)
+
+	dbCommand.BoolVar(&help1, "h", false, "Show the program usage message")
+	dbCommand.BoolVar(&help2, "help", false, "Show the program usage message")
+	dbCommand.Var(&args.Domains, "d", "Domain names separated by commas (can be used multiple times)")
+	dbCommand.BoolVar(&args.Options.ListEnumerations, "list", false, "Show the enumerations that include identified domains")
+	dbCommand.StringVar(&args.Filepaths.ConfigFile, "config", "", "Path to the INI configuration file. Additional details below")
+	dbCommand.StringVar(&args.Filepaths.Directory, "dir", "", "Path to the directory containing the graph database")
+	dbCommand.StringVar(&args.Filepaths.Domains, "df", "", "Path to a file providing root domain names")
+	dbCommand.StringVar(&args.Filepaths.Input, "i", "", "The Amass data operations JSON file")
```

Comment:
It's not obvious that this flag is used to import the data from the JSON file into the DB. Maybe change the text to:
`Import an Amass data operations JSON file`

Reply:
Agreed. It has been updated